share/eds/cache_test.go

```diff
@@ -0,0 +1,12 @@
+package eds
+
+import (
+	"testing"
+)
+
+func TestBlockstoreCache_Add(t *testing.T) {
+}
+
+func TestBlockstoreCache_Get(t *testing.T) {
+	t.Skip("not implemented")
+}
```

Comment:
?

Reply:
oh shit that file should have been deleted